src/contentmanagerside.cpp

```diff
@@ -92,7 +93,14 @@ void ContentManagerSide::setContentManager(ContentManager *contentManager)
 
 void ContentManagerSide::setCategories(QStringList categories)
 {
-    mp_categories->setSelections(categories, KiwixApp::instance()->getSettingsManager()->getCategoryList());
+    const auto lang = QLocale().name().split('_')[0].toStdString();
+    KiwixChoiceBox::SelectionList sList;
+    for (const auto& category : categories) {
+        const auto translation = QString::fromStdString(kiwix::translateBookCategory(lang, category.toStdString()));
+        sList.append({translation, category});
+    }
+
+    mp_categories->setSelections(sList, KiwixApp::instance()->getSettingsManager()->getCategoryList());
 }
```

Comment:
I think that after this change you can drop the `KiwixChoiceBox::setSelections(QStringList, SelectionList)` overload

Reply:
Done